Source/WebKit/Platform/spi/Cocoa/PDFKitSPI.h

```diff
@@ -98,29 +98,29 @@
 #endif
 @end
 
+@interface PDFDocument (Annotations)
+-(void)resetFormFields:(PDFActionResetForm *) action;
```

Comment:
(While we're moving this anyways)

```suggestion
- (void)resetFormFields:(PDFActionResetForm *)action;
```

Reply:
Will do